imagebuildah/stage_executor.go

```diff
@@ -1271,7 +1278,31 @@ func (s *StageExecutor) getBuildArgsResolvedForRun() string {
 			if inImage {
 				envs = append(envs, fmt.Sprintf("%s=%s", key, configuredEnvs[key]))
 			} else {
-				envs = append(envs, fmt.Sprintf("%s=%s", key, value))
+				// by default everything must be added to history
+				// this is configured to false only for special cases
```

Comment:
Please use punctuation in sentences and avoid lower-case only printing; I have a hard time brain-parsing the comments.

Reply:
Modified comments a bit.